src/bitcoin-cli.cpp

```diff
@@ -453,9 +454,29 @@ static UniValue ConnectAndCallRPC(BaseRequestHandler* rh, const std::string& str
     return response;
 }
 
+static CAmount AmountFromValue(const UniValue& value)
+{
+    CAmount amount{0};
+    if (!ParseFixedPoint(value.getValStr(), 8, &amount))
+        throw std::runtime_error("Invalid amount");
+    if (!MoneyRange(amount))
+        throw std::runtime_error("Amount out of range");
+    return amount;
+}
+
+static UniValue ValueFromAmount(const CAmount& amount)
+{
+    bool sign{amount < 0};
+    int64_t n_abs{sign ? -amount : amount};
+    int64_t quotient{n_abs / COIN};
+    int64_t remainder{n_abs % COIN};
+    return UniValue(UniValue::VNUM, strprintf("%s%d.%08d", sign ? "-" : "", quotient, remainder));
+}
```

Comment:
Can we get these into the `libbitcoin_util.a` module? I think this is the third copy...

Reply:
Yes, third one. Agreed, a good follow-up.

Reply:
Why not in this PR or in a base PR especially considering there aren't ACKs.

Reply:
I think that refactoring would best be done in a separate PR.